src/VSIX/VSIX.csproj

```diff
@@ -1,60 +1,62 @@
+    <SignAssembly>true</SignAssembly>
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <!-- Generic options copied from VSIX project templates -->
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <AssemblyName>Npgsql.VSIX</AssemblyName>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <CopyBuildOutputToOutputDirectory>true</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>true</CopyOutputSymbolsToOutputDirectory>
+    <GeneratePkgDefFile>true</GeneratePkgDefFile>
+    <IncludeAssemblyInVSIXContainer>true</IncludeAssemblyInVSIXContainer>
+    <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
+    <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
+    <OutputType>Library</OutputType>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <ProjectGuid>{ECFD8615-DEAD-4498-B262-85A4D0230229}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <RootNamespace>Npgsql.VSIX</RootNamespace>
-    <AssemblyName>Npgsql.VSIX</AssemblyName>
-    <TargetVsixContainerName>Npgsql.vsix</TargetVsixContainerName>
+    <SchemaVersion>2.0</SchemaVersion>
     <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <UseCodebase>true</UseCodebase>
-    <IncludeAssemblyInVSIXContainer>true</IncludeAssemblyInVSIXContainer>
-    <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
-    <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
-    <CopyBuildOutputToOutputDirectory>true</CopyBuildOutputToOutputDirectory>
-    <CopyOutputSymbolsToOutputDirectory>true</CopyOutputSymbolsToOutputDirectory>
+    <!-- unmatched -->
+    <NullableContextOptions>disable</NullableContextOptions>
+    <TargetVsixContainerName>Npgsql.vsix</TargetVsixContainerName>
+    <!-- Debug Parametwers -->
     <StartAction>Program</StartAction>
     <StartProgram Condition="'$(DevEnvDir)' != ''">$(DevEnvDir)devenv.exe</StartProgram>
     <StartArguments>/rootsuffix Exp</StartArguments>
-    <RuntimeIdentifiers>win</RuntimeIdentifiers>
-
-	<!-- Specific options added in Npgsql -->
-
-    <!-- See http://comealive.io/AppVeyor-Times-Out-VSIX/ for the following line -->
+    <!-- Specific options added in Npgsql -->
+    <!-- See: http://comealive.io/AppVeyor-Times-Out-VSIX/ for the following line -->
```

Comment:
FYI the Appveyor stuff can be removed (we no longer build on Appveyor).

Reply:
I didn't know..

Reply:
Removed